src/xds/server.rs

```diff
@@ -328,9 +329,9 @@ impl AggregatedControlPlaneDiscoveryService for ControlPlane {
                     let timeout = tokio::time::sleep(std::time::Duration::from_secs(1));
                     tokio::select! {
                         _ = timeout => {
-                            tracing::info!("sending discovery request");
-                            crate::xds::client::MdsStream::discovery_request_without_cache(&identifier, &mut requests, crate::xds::ResourceType::Cluster, &[])
-                                .map_err(|error| tonic::Status::internal(error.to_string()))?;
+                            // tracing::info!("sending discovery request");
```

Comment:
Was this meant to remain commented out?

Reply:
Yes, they aren't needed anymore, but I went ahead and cleaned it up